src/core/annotation.js

```diff
@@ -21,55 +21,7 @@
 'use strict';
 
 var DEFAULT_ICON_SIZE = 22; // px
-
-/**
- * @class
- * @alias AnnotationFactory
- */
-function AnnotationFactory() {}
-AnnotationFactory.prototype = /** @lends AnnotationFactory.prototype */ {
-  /**
-   * @param {XRef} xref
-   * @param {Object} ref
-   * @returns {Annotation}
-   */
-  create: function AnnotationFactory_create(xref, ref) {
-    var dict = xref.fetchIfRef(ref);
-    if (!isDict(dict)) {
-      return;
-    }
-
-    // Determine the annotation's subtype.
-    var subtype = dict.get('Subtype');
-    subtype = isName(subtype) ? subtype.name : '';
-
-    // Return the right annotation object based on the subtype and field type.
-    var parameters = {
-      dict: dict,
-      ref: ref
-    };
-
-    switch (subtype) {
-      case 'Link':
-        return new LinkAnnotation(parameters);
-
-      case 'Text':
-        return new TextAnnotation(parameters);
-
-      case 'Widget':
-        var fieldType = Util.getInheritableProperty(dict, 'FT');
-        if (isName(fieldType) && fieldType.name === 'Tx') {
-          return new TextWidgetAnnotation(parameters);
-        }
-        return new WidgetAnnotation(parameters);
-
-      default:
-        warn('Unimplemented annotation type "' + subtype + '", ' +
-             'falling back to base annotation');
-        return new Annotation(parameters);
-    }
-  }
-};
+var SUPPORTED_TYPES = ['Link', 'Text', 'Widget', 'Screen', 'Movie'];
```

Comment:
This change is incorrect. The `AnnotationFactory` should not be removed (causing the effect that the user above describes), but rather it should be adapted to add a case for `Screen` and `Movie`.